lib/parsers.js

```diff
@@ -94,7 +93,7 @@ module.exports = function(nanomatch, options) {
       if (!m) return;
       var val = m[0];
 
-      var isNegated = isOdd(val.length);
+      var isNegated = ((val.length%2)==1);
```

Comment:
Nitpicks:
 - remove the outer parentheses
 - put spaces before and after the `%` and the `==`
 - use strict equal `===`

Basically: `var isNegated = (val.length % 2) === 1;`